### PR TITLE
feat(explain): default explain with input tokenization

### PR DESCRIPTION
**Pull request by frascuchon**
This PR adds a default behaviour to `Pipeline.explain` method. 

If no explain was found/implemented, automatically generates an input tokenization as explain with no token attributions

Also, these changes exposes the number of steps (`n_steps`) for allow control attribution calculation (even deactivation if `n_steps` is less than 1.

### CODE REVIEW

File: src/biome/text/pipeline.py [hunk 1]
```diff
@@ -286,17 +288,24 @@ def predict(self, *args, **kwargs) -> Dict[str, numpy.ndarray]:
...
 
         The accepted input is dynamically calculated from head.input
 
+        Parameters
+        ----------
+
+        n_steps: int
+            The number of steps for token attribution calculation (if proceed).
+            If the number of steps is less than 1, the attributions will not be calculated
+
```
Comment (dcfidalgo) [inline src/biome/text/pipeline.py:302]: I think it should be possible to dynamically copy the docstring from the corresponding `TaskHead.explain_prediction()` or `_model.explain()` method. It is a bit strange that `n_steps` is the only parameter documented here, and the rest goes into the args and kwargs.

Comment (frascuchon) [reply src/biome/text/pipeline.py:302]: mmmm the explain/predict methods have dynamic arguments calculated directly from head.inputs. Normally these inputs correspond to `Head.featurize` method, but not always (the `RecordClassification` head set the pipeline input names in configuration.

File: src/biome/text/pipeline.py [hunk 2]
```diff
@@ -286,17 +288,24 @@ def predict(self, *args, **kwargs) -> Dict[str, numpy.ndarray]:
         """
         return self._model.predict(*args, **kwargs)
 
-    def explain(self, *args, **kwargs) -> Dict[str, Any]:
+    def explain(self, *args, n_steps: int = 5, **kwargs) -> Dict[str, Any]:
```
[the remark this answers is not in the document]
Comment (frascuchon) [reply src/biome/text/pipeline.py:291]: Setting a low level to `n_steps` we reduce performance problems with large models. @dvsrepo did some works and set 5 as a good trade-off  between performance and attribution calculation, but I'm ok with capcum number 50.

[the remark this answers is not in the document]
Comment (frascuchon) [reply src/biome/text/pipeline.py:291]: @dvsrepo something to say about that before merge the PR?

**Comment by dvsrepo** [comment]
All good on my side! Don't have a strong opinion on n_steps 50 as default
if it's more aligned with captum.

File: src/biome/text/pipeline.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (frascuchon) [reply src/biome/text/pipeline.py:291]: I'll keep to 5 as default for better performance

**Pull request merged**